hexapawn.py

```diff
@@ -47,6 +47,7 @@ def play_game():
         print('Winner is RED player')
     else:
         print('Winner is BLUE player')
+    state.draw_board()
```

Comment:
I like the idea of drawing the end state of a board after the end of a game! This seems like behavior that should been available from the beginning. :)